grapher/core/Grapher.tsx

```diff
@@ -501,14 +501,24 @@ export class Grapher
         )
     }
 
+    @computed private get tableAfterAuthorTimelineAndColumnFilter() {
+        return this.tableAfterAuthorTimelineFilter.keepOnlyColumns([
+            ...this.activeColumnSlugs,
+            this.inputTable.timeColumn.slug,
+            this.inputTable.entityNameSlug,
+            this.inputTable.entityIdColumn.slug,
+            this.inputTable.entityCodeColumn.slug,
```

Comment:
Not sure if it's necessary here, but can also include the `entityColor` slug, just to be safe?

Reply:
I think this chart has entity colors: https://tufte.owid.cloud/admin/test/embeds?ids=3361

And it seems they are getting dropped? Could also be that the databases are not in sync too.

Reply:
Nice catch!